### PR TITLE
feat: updated useXcmBridgeV3 and removed legacy files 

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* fix: updated useXcmBridgeV3
 *  get the api instance from apiFactory
* fix: removed legacy files related to XCM features 

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

### CODE REVIEW

File: src/v2/repositories/implementations/xcm/MoonbeamXcmRepository.ts [hunk 1]
```diff
@@ -16,12 +16,18 @@ import { Symbols } from 'src/v2/symbols';
...
 import { AbiItem } from 'web3-utils';
-import { capitalize } from './../../../../hooks/helper/common';
+import { capitalize } from 'src/hooks/helper/common';
 import { isValidEvmAddress } from 'src/config/web3';
 import { ethers } from 'ethers';
+import { AstarNativeToken } from 'src/v2/config/xcm/XcmRepositoryConfiguration';
 
 type ChainName = 'Moonriver' | 'Moonbeam';
 
+export const MOONBEAM_ASTAR_TOKEN_ID: AstarNativeToken = {
```
Comment (bobo-k2) [inline src/v2/repositories/implementations/xcm/MoonbeamXcmRepository.ts:26]: Removing `astarTokens `from constructor will break  `isAstarNativeToken `method which is used in the repo

**Review by bobo-k2 — CHANGES_REQUESTED** [review]

File: src/v2/repositories/implementations/xcm/MoonbeamXcmRepository.ts (hunk 1 above)
Comment (impelcrypto) [reply src/v2/repositories/implementations/xcm/MoonbeamXcmRepository.ts:26]: I have kept `astarTokens` [here](https://github.com/AstarNetwork/astar-apps/blob/729ac4047bdbc29b7489463cf0d47d2e5a4e5743/src/v2/repositories/implementations/xcm/MoonbeamXcmRepository.ts#L48)

Comment (bobo-k2) [reply src/v2/repositories/implementations/xcm/MoonbeamXcmRepository.ts:26]: I am sorry. I overlooked that

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Pull request merged**